### ISSUE TITLE
Add a developer share feature.

### ISSUE THREAD

**Issue by TXIuIENvZGU** [itchio/itch.io#521]
Where you can add multiple paypal links and select a percentage. Developer A gets 10% Developer B gets 80% Developer C gets 10%.

**Comment by hawkerm** [comment]
This is useful for collaborating with Artists too.  i.e. you should be able to split revenue to an account that may not be marked as a developer account which has its own page/projects and just be a user maybe?

**Comment by felixplesoianu** [comment]
I second this proposal. Will soon collaborate with another member of itch.io on my latest game, and it would be nice to just set a slider and forget about it. The alternative is fussing with PayPal myself after I get the money from sales, and that means extra charges, not to mention the risk that I'll send their share the "wrong" way and get myself banned. Thank you for considering it.

**Comment by barreytor** [comment]
Bumping this because I want this to happen, too. There's already multiple developers sharing money in bundles, so it'd be kinda like that only in the default settings of selling a game. Not actually sure of how hard it'd be, but there already seem to be enough groundwork parts for it to be little more than adding a "share of profits" slider to project admins. (At least that's how I see it)

**Comment by Calandiel** [comment]
Any plans or updates on this?

**Comment by fasterthanlime** [comment]
This has been mentioned internally recently, so it's still on our radar, but we're not able to give an ETA at this time.

**Comment by Ben-Gilmore** [comment]
@leafo, any update on this ticket? 

Still want to be able to manage payments this way.

**Comment by ParselyBunny** [comment]
This is so important. The logistics of revenue share are a huge obstacle to collaborative projects.

**Comment by skylorbeck** [comment]
+1 for this please

**Comment by cuttincows** [comment]
+1 on this, it would make proceed splits not contingent on one central dev

**Comment by elstiko** [comment]
+1 yes please, I want to collaborate and split profits

**Comment by Ratatatang** [comment]
+1, would make it so much less stressful to try to collaborate

**Comment by increpare** [comment]
As a usually solo-developer, I'd like to put a DRM version of a commercial collab game on itch (right not we only plan to make it available on steam), but having to double the number of invoices we have to fill out in perpetuity is somewhat prohibitive. ^^

**Comment by Yolwoocle** [comment]
+1. In our team we decided that I'd be the one to get the money and be responsible for splitting revenue. The logistics are a headache and will only get worse as we make more games with different teams composed of different people, not even accounting for bundles where I can't easily get the revenue obtained per game in the bundle. This is right now one of the biggest downsides of itch.io.

**Comment by rasgou** [comment]
Yes, please add this feature, add let us set it up per project, so each game could have a different % split set by the project owner (allowing even 0% for the owner for cases where it's meant as a team account, or more than zero if it's on the lead dev account). So each sale would automatically split after processor fees and itch cut, then each recipient pays their own taxes. Or essentially like selling a bundle (no idea how that works yet), but for one product.

This would help with e.g. student projects where people are still starting in gamedev and have to publish results somewhere, but are not strong enough financially to start companies yet. Specially since sending money internationally has its own whole set of implications, which would be each recipient's own problem to deal with when things are set up this way, simplifying everything.
